picard/tagger.py

```diff
@@ -322,8 +344,19 @@ def pipe_server(self):
         while self.pipe_handler.pipe_running:
             messages = [x for x in self.pipe_handler.read_from_pipe() if x not in IGNORED]
             if messages:
-                self.add_paths(messages)
-                self.bring_tagger_front()
+                self.load_to_picard(messages)
+
+    def load_to_picard(self, items):
+        parsed_items = ParseItemsToLoad(items)
+
+        if parsed_items.files:
+            self.add_paths(parsed_items.files)
+            self.bring_tagger_front()
+        if parsed_items.urls or parsed_items.mbids:
+            file_lookup = self.get_file_lookup()
+            for item in parsed_items.mbids | parsed_items.urls:
+                thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
+            self.bring_tagger_front()
```

Comment:
I was bothered by calling `bring_tagger_front()` twice, so I added `ParseItemsToLoad.__bool__()` in  https://github.com/skelly37/picard/pull/4
What do you think?

Reply:
Good idea, thanks, merged.